scan-payment/src/main/java/com/getbouncer/scan/payment/card/PaymentCardUtils.kt

```diff
@@ -217,14 +223,31 @@ fun String.lastFour(): String = lastFourFromPan(this)
  */
 @CheckResult
 internal fun getIssuerData(cardNumber: String): IssuerData? =
-    ISSUER_TABLE.firstOrNull { iinFromPan(cardNumber).toInt() in it.iinRange }
+    CUSTOM_ISSUER_TABLE.firstOrNull { iinFromPan(cardNumber).toInt() in it.iinRange }
+        ?: ISSUER_TABLE.firstOrNull { iinFromPan(cardNumber).toInt() in it.iinRange }
 
 /**
  * Get data for a given [CardIssuer].
  */
 @CheckResult
-private fun getIssuerData(issuer: CardIssuer): List<IssuerData> =
-    ISSUER_TABLE.filter { it.issuer == issuer }
+private fun getIssuerData(issuer: CardIssuer): List<IssuerData> {
+    val mutualIssuerData = CUSTOM_ISSUER_TABLE.filter { it.issuer == issuer }.toMutableList()
+    mutualIssuerData.addAll(ISSUER_TABLE.filter { it.issuer == issuer })
+
+    return mutualIssuerData.toList()
+}
```

Comment:
```suggestion
private fun getIssuerData(issuer: CardIssuer): List<IssuerData> =
    (CUSTOM_ISSUER_TABLE + ISSUER_TABLE).filter { it.issuer == issuer }
```

Reply:
Was looking for solution like this, but didn't find any matching functions, the answer was way easier than I thought.